packages/core/src/config/utils.ts

```diff
@@ -198,47 +199,47 @@ export function mergeExtends(rulesConfList: ResolvedStyleguideConfig[]) {
       );
     }
 
-    Object.assign(result.rules, rulesConf.rules);
-    Object.assign(result.oas2Rules, rulesConf.oas2Rules);
-    assignExisting(result.oas2Rules, rulesConf.rules || {});
```

Comment:
Why removed `|| {}`? Either remove it for all cases or add it back here. Actually, I see you've added a [check](https://github.com/Redocly/redocly-cli/pull/1811/files#diff-6552eacdc0f4378c00e24a96877df8352eeb0db006852dfdcf511f59d9b781aeR217) for an empty object, so the fallback could be removed.

Reply:
oh, had to be for all cases, thanks. removed everywhere